test/watch.js

```diff
@@ -285,3 +285,14 @@ testCb("--watch doesn't exit on CssSyntaxError", (t) => {
   // Timeout:
   setTimeout(() => t.end('test timeout'), 50000)
 })
+
+testCb('--watch does exit on closing stdin (Ctrl-D/EOF)', (t) => {
+  t.plan(0)
+
+  const cp = spawn(
+    `node ${path.resolve('bin/postcss')} -o output.css -w --no-map`,
+    { shell: true }
+  )
+  cp.on('exit', () => t.end())
```

Comment:
Should add `cp.on('error', t.end)` above this line.

Reply:
Thanks, fixed!